### ISSUE TITLE
ServiceControl Service Stops - version 1.6.3 from 1.4

### ISSUE THREAD

**Issue by cbrown11** [Particular/ServiceControl#571]
Since upgrading to version 1.6.3 from 1.4, the service keeps stopping with the following error below. Seems not to find the error queue, even though it does exists

```
2015-10-26 09:48:52.1101|8|Fatal|NServiceBus|Error Import cannot start
NServiceBus.Unicast.Queuing.QueueNotFoundException: Failed to send message to address: [#######.dv4.crpt.qir.error.log@#####] ---> System.Messaging.MessageQueueException: The queue does not exist or you do not have sufficient permissions to perform the operation.
   at System.Messaging.MessageQueue.MQCacheableInfo.get_WriteHandle()
   at System.Messaging.MessageQueue.StaleSafeSendMessage(MQPROPS properties, IntPtr transaction)
   at System.Messaging.MessageQueue.SendInternal(Object obj, MessageQueueTransaction internalTransaction, MessageQueueTransactionType transactionType)
   at NServiceBus.Transports.Msmq.MsmqMessageSender.Send(TransportMessage message, Address address) in c:\BuildAgent\work\1b05a2fea6e4cd32\src\NServiceBus.Core\Transports\Msmq\MsmqMessageSender.cs:line 59
   --- End of inner exception stack trace ---
   at NServiceBus.Transports.Msmq.MsmqMessageSender.Send(TransportMessage message, Address address) in c:\BuildAgent\work\1b05a2fea6e4cd32\src\NServiceBus.Core\Transports\Msmq\MsmqMessageSender.cs:line 75
   at ServiceControl.Operations.ErrorQueueImport.TerminateIfForwardingQueueNotWritable() in C:\BuildAgent\work\7189a56f9f44affc\src\ServiceControl\Operations\ErrorQueueImport.cs:line 97
```

Service Config
```
<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<configuration>
  <appSettings>
    <add key="ServiceControl/LogPath" value="d:\logs" />
    <add key="ServiceControl/Name" value="Particular.ServiceControl.DV4" />    
    <add key="ServiceBus/AuditQueue" value="#####.dv4.crpt.qir.audit" />
    <add key="ServiceBus/ErrorQueue" value="#####.dv4.crpt.qir.error" />
    <add key="ServiceControl/Hostname" value="#####" />
    <add key="ServiceControl/Port" value="33333" />
    <add key="ServiceControl/ExposeRavenDB" value="false" />
    <add key="ServiceControl/ForwardAuditMessages" value="false" />    
    <add key="NServiceBus/LicensePath" value="C:\NServiceBus\License\License.xml" />
  </appSettings>
</configuration>
```

### Steps to repro
1. UnInstall ServiceControl 
2. lnstall ServiceControl (config own error queue location) and start
3. UnInstall Endpoints with a complete clear down including ravenDb and MSMQ messages
4. ReInstall Endpoints 
5. Trying starting ServiceControl and then stops with the error above

### Work around 
Is to reinstall the ServiceControl again 
or 
change the order and install the endpoints before the ServiceControl is installed.

**Comment by gbiellem** [comment]
Hi  @cbrown11 

We've  hit this one before and we are addressing it in 1.7. ( which we are prepping for release now)

Basically what is happening is you have changed the value of error queue and this has the side effect that it also changes the expected queue name for the errorlog queue unless the name is specified in "ServiceBus/ErrorLogQueue".   The same is true for the audit vs auditlog queue although in your case this doesn't apply since you have audit forwarding turned off.  

As ServiceControl only creates queues at installation time not run time you would have to create the expected queue manually or run "servicecontrol --Install" from the command line to force queue creation in ServiceControl 1.6

The  management app in ServiceControl 1,7 will call queue creaion when the configuration is changed through the UI  so this problem will be resolved.

**Comment by gbiellem** [comment]
Fixed in 1.7
